resources/views/admin/followerfollowinglist.blade.php

```diff
@@ -0,0 +1,71 @@
+@extends('layouts.app')
+
+@section('title', 'Admin: Follower/Folloing List')
+
+@section('content')
+    <div class="justify-content-center rounded border" style="min-width:696px">
+        <h1 class="ps-3 mt-2">Follower/Folloing List</h1>
```

Comment:
You might have misspelled "Following" also in line #3

Reply:
I fixed misspelled "Following" in [47d704b](https://github.com/kredo-school/16th_rankshty/pull/13/commits/47d704badd7800fb09c501b2eeb2457bedc29e4a) and [dfdb030](https://github.com/kredo-school/16th_rankshty/pull/13/commits/dfdb030c1cdcedc707a0049e9e38a5d6ab8131e2).